app/models/target.rb

```diff
@@ -73,6 +73,12 @@ def self.valid_roles
   VISIBILITY_ARCHIVED = 'archived'
```

Comment:
Document target checklist schema at the top of this file. For a mini example, see [models/user.rb](https://github.com/SVdotCO/pupilfirst/blob/master/app/models/user.rb).